### DISCUSSION TITLE
at one point wasn't there a plugin that grayed out all text except matches when using f/F searching?

### DISCUSSION THREAD

**Discussion by jrnxf** [LazyVim/LazyVim#310] [Q&A] [answered]
I swear when I first downloaded LazyVim and I would use f/F to search, it would gray out all the text in front of or behind my cursor that didn't match the search params. Does anyone else remember that or how it worked? I'd like to get it back if possible!

**Comment by jrnxf** [comment, marked as answer]
Ah! Found it. It's just highlight group styling via Leap. eg:

```lua
LeapBackdrop = { link = "Comment" },
LeapLabelPrimary = { bg = terafox.sel0, fg = colors.off_white_text },
LeapLabelSecondary = { bg = terafox.sel1, fg = colors.off_white_text },
```

**Reply by folke** [reply]
Ah, yes, TokyoNight supports leap/flit, but probably not all colorschemes indeed
